### DISCUSSION TITLE
apollotracing handler does not support federated tracing

### DISCUSSION THREAD

**Discussion by wyattEpp** [99designs/gqlgen#1923] [Ideas]
`apollotracing` does not support federated tracing as outlined in [this section of Apollo docs](https://www.apollographql.com/docs/federation/metrics/#how-tracing-data-is-exposed-from-a-subgraph). The current version expects an `ftv1` field with tracing data, not a `tracing` field.

### What happened?
Despite having Apollo tracing enabled in Go subgraphs built with gqlgen, trace information was not picked up by our Apollo Server GraphQL API gateway and reported to Apollo Studio. 

### What did you expect?
Tracing data to be reported under the `ftv1` extension field, such that it could be recognized by Apollo Server.

### Minimal graphql.schema and models to reproduce
Any schema with the apollotracing handler enabled as shown here https://github.com/99designs/gqlgen/blob/master/graphql/handler/apollotracing/tracer_test.go#L33-L35.

### versions
 - `gqlgen version` 0.13.0
 - `go version` 1.15.6

**Comment by mlaflamm** [comment]
For those who missed it, there was some progress on this recently #2132
